### PR TITLE
CORE-13266: delete deprecated HttpService

### PR DESCRIPTION
**Pull request by BenYip123**
- Delete deprecated `HttpService`, `HttpRpcService` and `HttpServiceUser`
- Delete fuel and json dependencies

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -29,13 +29,7 @@ dependencies {
...
     implementation "org.apache.commons:commons-lang3:${commonsLangVersion}"
     implementation "org.yaml:snakeyaml:${snakeyamlVersion}"
-    implementation "com.github.kittinunf.fuel:fuel:${fuelVersion}"
-    implementation "com.github.kittinunf.fuel:fuel-json:${fuelVersion}"
-    constraints {
-        implementation("org.json:json:${jsonVersion}") {
-            because("Required until fuel-json updates it's internal version of org.json, not fixed as of fuel version 2.3.1")
-        }
-    }
+    implementation "org.json:json:${jsonVersion}"
```
Comment (vkolomeyko) [inline app/build.gradle]: What happens if we delete:
```
implementation "org.json:json:${jsonVersion}"
```
?
(if this works, we can delete `jsonVersion` constant)

Also, I think we can delete `fuelVersion` Gradle constant.

Comment (BenYip123) [reply app/build.gradle]: Looks like it is ok without that dependency

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**